packages/daemon/src/services/index.ts

```diff
@@ -166,7 +168,7 @@ export const handleVertexAccepted = async (context: Context, _event: Event) => {
   try {
     const fullNodeEvent = context.event as FullNodeEvent;
     const now = getUnixTimestamp();
-    const { NEW_TX_SQS, PUSH_NOTIFICATION_ENABLED } = getConfig();
+    const { PUSH_NOTIFICATION_ENABLED } = getConfig();
```

Comment:
Was this moved for any specific reason or is it the same? Before we were using `getConfig()` and now it's a direct import

Reply:
It should be! Thanks